index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Quick startup</title>
+</head>
+
+<body>
+    <!--This wraps on my page together-->
+    <div class="container">
+        <header class="header-page">
+            <!--header contents goes here-->
+
+            <nav>
+                <!--This is my Logo-->
+                <h3>Solorics</h3>
+
+                <!--Toggle button goes here-->
+                <img src="images/Union.png" alt="Toggle_Image_Button" class="toggle">
+            </nav>
+        </header>
+        <main class="main-page">
+            <!--main contents goes here-->
+            <div>
+                <h3>Opeyemi</h3>
+                <h4>Software Developer</h4>
+                <h5>I can help you build a product , feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.</h5>
```

Comment:
- [ ] Let's change this `<h5>` tag to `<p>`.  
This way it will be semantically correct and more accessible for screen readers etc.   Cheers 👍